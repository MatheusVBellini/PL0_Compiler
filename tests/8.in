VAR flag;
PROCEDURE setTrue;
BEGIN
  flag := 1;
END;
BEGIN
  CALL setTrue;
END.
