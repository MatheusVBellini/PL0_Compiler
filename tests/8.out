VAR, VAR
flag, ident
;, simbolo_ponto_virgula
PROCEDURE, PROCEDURE
setTrue, ident
;, simbolo_ponto_virgula
BEGIN, BEGIN
flag, ident
:=, simbolo_atribuicao
1, numero
;, simbolo_ponto_virgula
END, END
;, simbolo_ponto_virgula
BEGIN, BEGIN
CALL, CALL
setTrue, ident
;, simbolo_ponto_virgula
END, END
., simbolo_ponto
